### PR TITLE
test: locked_wallet, skip default fee estimation

### PR DESCRIPTION
**Pull request by furszy**
Coming from https://github.com/bitcoin/bitcoin/pull/28139#discussion_r1284563239.

No test case in this file is meant to exercise fee estimation. All default wallets have a
custom tx fee set [here](https://github.com/bitcoin/bitcoin/blob/b7138252ace6d21476964774e094ed1143cd7a1c/test/functional/wallet_fundrawtransaction.py#L100). The only one missing is the one created for `locked_wallet`.

### CODE REVIEW

**Review by maflcko — APPROVED** [review]

File: test/functional/wallet_fundrawtransaction.py [hunk 1]
```diff
@@ -599,8 +601,12 @@ def test_locked_wallet(self):
 
         # Choose input
         inputs = wallet.listunspent()
-        # Deduce fee to produce a changeless transaction
-        value = inputs[0]["amount"] - Decimal("0.00002200")
+
+        # Deduce exact fee to produce a changeless transaction
+        expected_tx_size = 110  # Total tx size: 110 vbytes, p2wpkh -> p2wpkh. Input 68 vbytes + rest of tx is 42 vbytes.
+        fee_rate = self.min_relay_tx_fee / 1000  # btc/vbytes
+        value = inputs[0]["amount"] - (fee_rate * expected_tx_size)
```
Comment (theStack) [inline test/functional/wallet_fundrawtransaction.py:608]: nit: could use the `get_fee` helper (from test_framework.util) for not having to bother with manual fee-rate unit coversion and fee calculation:
```suggestion
        value = inputs[0]["amount"] - get_fee(expected_tx_size, self.min_relay_tx_fee)
```

Comment (furszy) [reply test/functional/wallet_fundrawtransaction.py]: sure, pushed. Thanks.
would be nice to have a document somewhere mentioning the existence of these utility functions.

**Comment by furszy** [comment]
Updated per feedback. Thanks theStack.

Tiny change. Instead of calculating the fee manually, the code now uses
the `get_fee()` util function.

**Review by theStack — APPROVED** [review]
ACK 5364dd8666ae1afa35536f9b4fc0170eeaf37888

**Pull request merged**